pkg/utils/common.go

```diff
@@ -616,7 +617,7 @@ func NewLogger(
 		},
 		LogLevel: logLevel,
 	}
-	if logToStdout {
+	if sigAggFlags.AggregatorLogToStdout {
```

Comment:
Isn't the log level set either way?

Reply:
display level is specific to stdout, if not set, no logs to stdout